R/pkg/R/mllib.R

```diff
@@ -119,7 +121,7 @@ NULL
 #' @note spark.glm since 2.0.0
 #' @seealso \link{glm}, \link{read.ml}
 setMethod("spark.glm", signature(data = "SparkDataFrame", formula = "formula"),
-          function(data, formula, family = gaussian, tol = 1e-6, maxIter = 25) {
+          function(data, formula, family = gaussian, weightCol = NULL, tol = 1e-6, maxIter = 25) {
```

Comment:
you might not want to add a parameter in the middle of the list. if someone has existing code calling this function in parameter order it might misalign